Tests/ScipioKitTests/CacheSystemTests.swift

```diff
@@ -1,31 +1,22 @@
 import Foundation
 @testable import ScipioKit
 import XCTest
+import Basics
+
+private let fixturePath = URL(fileURLWithPath: #filePath)
+    .deletingLastPathComponent()
+    .appendingPathComponent("Resources")
+    .appendingPathComponent("Fixtures")
+
+final class CacheSystemTests: XCTestCase {
 
-final class ClangCheckerTests: XCTestCase {
-    private let clangVersion = """
-Apple clang version 14.0.0 (clang-1400.0.29.102)
-Target: arm64-apple-darwin21.6.0
-Thread model: posix
-InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
-"""
     private let customModuleMap = """
     framework module MyTarget {
         umbrella header "umbrella.h"
         export *
     }
     """
 
-    func testParseClangVersion() async throws {
-        let hook = { arguments in
-            XCTAssertEqual(arguments, ["/usr/bin/xcrun", "clang", "--version"])
-            return StubbableExecutorResult(arguments: arguments, success: self.clangVersion)
-        }
-        let clangParser = ClangChecker(executor: StubbableExecutor(executeHook: hook))
-        let version = try await clangParser.fetchClangVersion()
-        XCTAssertEqual(version, "clang-1400.0.29.102")
-    }
```

Comment:
I hope it's fine that I extracted the Clang version checking logic into a separate test case.